### ISSUE TITLE
[GlobalStep] [Chrome] For Atomic sites, "PHP error in Ajax response" flashes while placing an order from Checkout page.

### ISSUE THREAD

**Issue by gglobalstep** [woothemes/woocommerce#30557]
## Bug Description:
For Atomic sites, "PHP error in Ajax response" flashes while placing a order from Checkout page.

## Environment:
Woocommerce Version : WooCommerce 5.7.0 Beta-1
WordPress version: v5.8

PC:
Windows 10, Mac 10.14.6
Chrome(Version 92.0.4515.131) 

## Steps To Reproduce:
1. Go to automic site.
1. Login with valid credentials.
1. Install and activate all the required plugins.
1. Complete the setup wizard.
1. Add or Import some Products.
1. Go to Store page.
1. Add any product to the "Cart" and go to "Checkout" page.
1. Enter all the billing details and click on "Place order".
1. Observe that "PHP error in Ajax response" error flashes for few seconds on browser console.

## Actual Result:
For Atomic sites, "PHP error in Ajax response" flashes while placing a order from Checkout page.

## Expected Result:
User should be able to Place an order without any errors.

## Screenshot:

![#30557](https://user-images.githubusercontent.com/41110392/130473207-1f833faf-f0a3-4a0e-9e2d-f9b3c92174eb.png)

*Isolating the problem (mark completed items with an [x]):*
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

<details>

`
### WordPress Environment ###

WC Version: 5.7.0
REST API Version: ✔ 5.7.0
WC Blocks Version: ✔ 5.7.0
Action Scheduler Version: ✔ 3.2.1
WC Admin Version: ✔ 2.6.0-rc.2
Log Directory Writable: ✔
WP Version: 5.8
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: ✔

### Server Environment ###

Server Info: nginx
PHP Version: 8.0.9
PHP Post Max Size: 2 GB
PHP Time Limit: 1200
PHP Max Input Vars: 6144
cURL Version: 7.78.0
OpenSSL/1.1.1d

SUHOSIN Installed: –
MySQL Version: 5.5.5-10.4.20-MariaDB-log
Max Upload Size: 2 GB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 5.7.0
WC Database Prefix: wp_
Total Database Size: 9.28MB
Database Data Size: 6.67MB
Database Index Size: 2.61MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 0.02MB + Index: 0.13MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_gla_budget_recommendations: Data: 0.22MB + Index: 0.14MB + Engine InnoDB
wp_gla_merchant_issues: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_gla_shipping_rates: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_gla_shipping_times: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailchimp_carts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailchimp_jobs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_custom_fields: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_dynamic_segment_filters: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_feature_flags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_forms: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_log: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_mapping_to_external_entities: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletters: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_newsletter_links: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_mailpoet_newsletter_option: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_option_fields: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_posts: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_segment: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_templates: Data: 2.52MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_scheduled_tasks: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_scheduled_task_subscribers: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_segments: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_sending_queues: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_settings: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_statistics_clicks: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_mailpoet_statistics_forms: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_statistics_newsletters: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_statistics_opens: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_mailpoet_statistics_unsubscribes: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_mailpoet_statistics_woocommerce_purchases: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_mailpoet_stats_notifications: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_subscribers: Data: 0.02MB + Index: 0.13MB + Engine InnoDB
wp_mailpoet_subscriber_custom_field: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_subscriber_ips: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_subscriber_segment: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_user_flags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_options: Data: 2.16MB + Index: 0.03MB + Engine InnoDB
wp_postmeta: Data: 0.16MB + Index: 0.14MB + Engine InnoDB
wp_posts: Data: 0.06MB + Index: 0.06MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB

### Post Type Counts ###

attachment: 23
mailpoet_page: 1
page: 7
post: 2
product: 18
product_variation: 7
shop_coupon: 3
shop_order: 16
shop_order_refund: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (8) ###

Query Monitor: by John Blackbourn – 3.7.1
Akismet Anti-Spam: by Automattic – 4.1.10
Google Listings and Ads: by WooCommerce – 1.4.1
Jetpack: by Automattic – 10.0
Mailchimp for WooCommerce: by Mailchimp – 2.5.2
MailPoet 3 (New): by MailPoet – 3.67.0
WooCommerce Shipping & Tax: by WooCommerce – 1.25.18
WooCommerce: by Automattic – 5.7.0-beta.1

### Dropin Plugins (3) ###

advanced-cache.php: advanced-cache.php
db.php: Query Monitor Database Class
object-cache.php: Memcached

### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔

### WC Pages ###

Shop base: #6 - /shop/
Cart: #7 - /cart/
Checkout: #8 - /checkout/
My account: #9 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Storefront
Version: 3.8.1
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Action Scheduler ###

Complete: 46
Oldest: 2021-08-23 12:34:19 +0000
Newest: 2021-08-23 14:25:40 +0000

### Status report information ###

Generated at: 2021-08-23 14:51:11 +00:00
`

</details>

**Comment by zhongruige** [comment]
Hi @gglobalstep,

Thank you for submitting the issue. However, I can’t reproduce it using the steps you provided. Everything is working as expected on my end. 

* I set up a new WooCommerce site and imported the sample products
* I went to the store and placed an order and was able to complete it without any logs in the console.
* I tested this both on Chrome and Firefox:

<img width="2552" alt="Screen Shot 2021-08-23 at 14 27 46" src="https://user-images.githubusercontent.com/71906536/130514889-0964a0af-5082-4ac0-943f-676b751f7259.png">

<img width="1437" alt="Screen Shot 2021-08-23 at 14 27 13" src="https://user-images.githubusercontent.com/71906536/130514916-8ade8ae6-0402-44b2-b709-51a02cbc7b95.png">

Please provide us with more details about the issue which may help us to evaluate it further. 

* What other plugins do you have installed? For example, were the recommended plugins as part of the onboarding wizard installed?
* Does the order go through as expected or does this error block placing an order?

Thank you in advance!

**Comment by gglobalstep** [comment]
> Please provide us with more details about the issue which may help us to evaluate it further.
> 
> * What other plugins do you have installed? For example, were the recommended plugins as part of the onboarding wizard installed?
> * Does the order go through as expected or does this error block placing an order?
> 
> Thank you in advance!

We have observed this issue only with the Automic site. Could you please check once on the Automic site?

**Comment by rodelgc** [comment]
Hi @gglobalstep,

Thank you very much for logging the issue. I noticed that the error message you encountered had something to do with the plugin "MailPoet". However, I still couldn't replicate the issue when I tried it in a different Atomic site that has all the extensions suggested in the onboarding wizard (including MailPoet).

<img width="1379" alt="Screen Shot 2021-08-26 at 6 12 39 PM" src="https://user-images.githubusercontent.com/4509348/130945227-bcb64d63-ca14-48ce-be47-a3af6016ff8f.png">

So I'm guessing there must be something specific in the Atomic site you tested that could have caused the error message.

For the meantime, I'm adding a `needs feedback` label in case you happen to discover a key step to replicate this issue. Thanks again!
